doc/conf.py

```diff
@@ -132,19 +117,20 @@
 # https://github.com/readthedocs/sphinx_rtd_theme/issues/766#issuecomment-513852197
 html4_writer = True
 
-# Read the docs style:
-if os.environ.get("READTHEDOCS") != "True":
-    try:
-        import sphinx_rtd_theme
-    except ImportError:
-        pass  # assume we have sphinx >= 1.3
-    else:
-        html_theme_path = [sphinx_rtd_theme.get_html_theme_path()]
-    html_theme = "sphinx_rtd_theme"
-
-
-def setup(app):
-    app.add_css_file("fix_rtd.css")
+html_theme = "pymc_sphinx_theme"
+html_theme_options = {
+    "use_search_override": False,
+    "logo": {
+        "text": "PyTensor",
```

Comment:
Shall we use `images/pytensor_logo.svg` here?

Reply:
wasn't aware we had a logo image, will update to use it. It is a bit different from this change